master-test-cases.yml

```diff
@@ -378,14 +382,11 @@ body:
6TekzM7UUWNfGqW5zMIA6TCEVFrxTcP2TkylhYV9LRPf1LywtgcW21xHY/RAFqN8vbJaUoMEoD9BhLcDTJym3dpLMOHH2ExAyeFWIJZ6XDut/0GqN3QSIaOTdhbQoJSg3WRnP+GFyVQFSQLbskOsyA1PxxUwyA+qEU6450XkLoB/EZf8juyYM4CZaCx+e9aVicuUbso3qqNcD0v2AY9QtIIRG+nNJJwwmRCOi29JSHFg2tqmuvLGJPO2952LqsRlV+DjoKdh1R5mdMjaWhH6ST2OuSxHQS/+mLiy3jNu5E2UB+7xYnQXQBfL8Fctv9AAa2D35uCMWE8QaI3vLh0OrjRDrDR7JFkZl0UgQfsGfUsR4PT4udwoFa1RSHEOrKyI81oqtKN20yx/I4XM0w1zlNl5mUe/DktESA7rAceZDOilPRfUUBq6DDJdrxng3hNLqPieOX4v8tAMJvUh2d8quxyZMlGoY+RDoFxMrnDfCnyYrHN4TWZ4Lbt3lPO2qeh0lfRlh/8bSOJqxvKt+vgWjuxq2kIi7B3IUxh1On0F0xa+RK0LlovqnIL/9hd75WsXbkm6qPeRjGNyzgDmFGX8p2ku2S+y16OZKbR/K9Xu/D5S+HHldM85fD+wUa01+T4O3dk0TKFLAklUCBUSGY836c1YtD6HJj+1C51Pfml2X2Fr80ia5Ru4OptGb1NlFCa9daQG8P71hyM9k0ucF5+N+A5hRZfEKTPpufrXFQjhPf1xrRjNY/fhejj5Ve6KAl1uqitcpa8vT6QuN9xiLqwphTmZPuVi0uLRU3+D5acSrF+W/IGEWgz7wwgmq7qeiHUMD+py4K8ufAOfZrc+7CZs1Jqm8+5Bvki4aVf0oZbPvis9+h4qvSwsFqX2m22XsLZLWa65kRYhjh5s483irLZ4QWAgurV2yFCQleKkcbPgvATLM18pGzUOL91SdYyg0gkfedLpdwRlbGRq6Ojk42WUZshoDeVeBfeIgGnqAnvW9DM22U8Zo0OXSKMGnpCe7/70CHmkGoKCPnX47b69tnSQa8BDEK25JgQVKPrY0F/QJ/mZ6Q0kb+JF07mXCgti5nWyvn3RYP4c3rKfrCrcKnS7Pt2s8zok2B2vZKx1MpNrKgJILH2X8Z3yX1he3CH+psOS1WBMXlzI9HWMI/c4vo0oyOTKN9jkYTm7Eg1hYlxkdtbY/TOwE4bEmBs63Riz/ok/MVBhvPbNeo+QMIDRJ7m2tLaDkBteZyZ/MkamKCVQnTUaHRS2t3mJJbq7OVTsgS5rSLz5HkWcqZHC/hPPius02UPswDwUCKqGirB11mkthH6YYS4DqWXZFOeVt4D8X6iHMwuSjagzWAcu/Jkvl5k7s29g/k1m3LHnlprOEsHKzMRTLYNsYMHuKBl62A+SToKLyNq4e3BBzSzgTyizFVd1larlmTrigGuh3M/NR0/XEtfRYG8g8fRMgFsMB8clN7vc0fCOGvUmL32l1tz6mezfSDEgGX0VTErkVOCX/brtf3E3LhpB6RqbWhnbuDMTEEzefm4XmiMdLummV82OZeCZekhYXxo1nFAz5lPttHG5xHwjvMhUZeTW4AsZy4Hp8BiKvqMKM7cwh0mV0OoaAs0jbVoL8NEzJewdK3iWfMiGI2NQ4eNzIswYfVqehKy7vFgSr8IjcN+KVni6ET8caqlglvnDjGqu8b0UEdzzzlv1rSsUsv3dfVsSxQ8Jw70PL5GLS8wUwHkgVqZGZZ6AlIEuv7Ed7W6IEDUjNM4PgydCWaJoY+Ic33KQGhm82Jhg4WMz8mrSg7YZE/s4bYEktCWyH9G1mdjOWZdr75+z5NZRQqbkKQwYHwTSyxLDmS//YJrrJqjigZ9/8rCMpvu/X4aYUYR2XB1Z4sxz0xVmRO4GGa2ccN+5Uf4y/RBWtkN17dy0CvvurzUbLNsvX+SIM0pWCn3WxxWiLcpG76RFxuraLyiA2DaNdZsJtnMMgwnHFqJ2MqJofWAqQRdnpvBbf/MK7BDbX3B3L7EGh4xrtpd60bzm6v7RYDN56BCMMHyOohwoK4NbzVU1L+kzaJ0/rpHLCZ7BOud+aFPzTY25EbYW7NMv02XUcrBRdaoYDHwd4D6wJMnQt675YQvQRNDQQ86piwkBgg3bOcGsgQOCqDdIck03Y/nxzqo3Dvomyf8JKsMNmQPwUMLlGhD3F+b7dKXUkebABClwrsA+UJ5PkwsIsQ/Y+fNmjsLWAzdgomIWwsRB3F6KYbLZonDipR4sjCsiuPexX5FI0dFk7zbhwzH3oebxAApRtBNRsqtVsSbPOyN8aAwAijheYFrCRndoktXuTZ9fX8hevz+zTPw5Mdayg2rh/rnckmNJVtsTZ1vbqHE6IhLMqy+3UTQ0RsH39OTsD2t6xbBudZQEQ9LVhWEBye8A7ZsV0lR2dCIyQ6bnP++GNQ3BFP22txV/tQEY/56Q9xdOn6iqGFMtM0ruVK75rwMEvWI2ZxK7Gg05RKyNksqXaH96ax343/cuOPtbmqa6nnjF0+iNr708Asfjb78S1V49w+z5kCjeK1r+tSWBkz7dM7fwxv2Esd4/vYWZqYHx+yPy3Z2TpaQQ/rXQN6apON+AzXQ8zJMAejylFu6cT5cUPoWkx3e/VlzTLVfHZ2dVqb4zyWMfCz4kwOWZRGYVlM1j5wbpEgcb4elI4e5hAUJYydGtu5UWnDxZvA9bqmexB7abnJfoq/fg35ye/CZlJEsZamy1GzFHzY1x+wnJsUbh++gKE4RDZsFzI2xqWwRY+X3EEczYOPD88+CpnZ6hh9bOcASgWJTMcD9PrtFjtvGTvVtjS7nNsejlYhunuaxyXoChLCGGAt67mTIH3jLHgNI3DCDPFaRPeVefQQrUsSeiGjsKIcaGuZoYJD42ubF2/IIiVu1s02SHUHpHxH2PligypbgB2iAAXkpKKFIfsEdhqcqiu3NgNBB2D747o8jS6ypn9eu640nYD7lYGNUGcfGyjaDVmdCQ=="'
```

Comment:
this test case is massive because we only ever flush output on 4K and also wanted to make sure there's multiple chinks in the response.

Reply:
conjure-undertow flushes at ~16k

Reply:
interesting, I think since this is a server test case we have to make sure that this server implementation flushes it. probably need to add a test for that

Reply:
I guess I can always make it bigger